File: LeetCode/LC-1491-Avg-Salary-Excluding-Min-Max/Solution.py
class Solution:
    def average(self, salary: List[int]) -> float:
        minS,maxS = 10000000 , -1
        sumS = 0

        for i in salary:
            minS = i if i < minS else minS
            maxS = i if i > maxS else maxS
            sumS += i
        
        sumS -= (maxS + minS)

        return sumS/(len(salary) - 2);

File: LeetCode/LC-1491-Avg-Salary-Excluding-Min-Max/Solution.java
class Solution {
    public double average(int[] salary) {
        int min = Integer.MAX_VALUE , max = Integer.MIN_VALUE;
        int sum = 0;

        for(int i : salary){
            min = i < min ? i : min;
            max = i > max ? i : max;
            sum+=i;
        }
        sum -= (min + max);

        double res = (double)sum/(salary.length - 2);
        return res;
    }
}